deebot_client/commands/xml/charge_state.py

```diff
@@ -0,0 +1,57 @@
+"""Error commands."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from deebot_client.events import StateEvent
+from deebot_client.message import HandlingResult
+from deebot_client.models import State
+
+from .common import XmlCommandWithMessageHandling
+
+if TYPE_CHECKING:
+    from xml.etree.ElementTree import Element
+
+    from deebot_client.event_bus import EventBus
+
+
+class GetChargeState(XmlCommandWithMessageHandling):
+    """Get pos command."""
+
+    name = "GetChargeState"
+
+
+    @classmethod
+    def _handle_xml(cls, event_bus: EventBus, xml: Element) -> HandlingResult:
+        """Handle xml message and notify the correct event subscribers.
+
+        :return: A message response
+        """
+
+        # examples:
+        # <ctl ret='ok'><charge type='SlotCharging' g='0'/></ctl>
+        # <ctl ret='ok'><charge type='Idle' g='0'/></ctl>
+        # <ctl ret='ok'><charge type='Going' g='0'/></ctl>
```

Comment:
```suggestion
```

Test are required for new commands. Please add some instead of adding comments as examples